source/page-xero.html.erb

```diff
@@ -70,14 +70,13 @@ section_class: 'section_home'
     </div>
   </section>
 
-  <% if locale_obj[:xero_cashbook] %>
+  <% if true || locale_obj[:xero_cashbook] %>
```

Comment:
Wouldn't this always evaluate to `true`? I.e `true || false === true`?

Reply:
No, this was me debugging...thank you!

```suggestion
  <% if locale_obj[:xero_cashbook] %>
```